src/ontology/mondo-ingest.Makefile

```diff
@@ -353,15 +357,27 @@ signature_reports: $(ALL_MIRROR_SIGNTAURE_REPORTS) $(ALL_COMPONENT_SIGNTAURE_REP
 #############################
 #### Lexical matching #######
 #############################
+# Official Mondo SSSOM Mappings
+# - Doeesn't include: broad mappings
+tmp/mondo.sssom.tsv:
+	wget http://purl.obolibrary.org/obo/mondo/mappings/mondo.sssom.tsv -O $@
+
+tmp/mondo.sssom.ttl: tmp/mondo.sssom.tsv
+	sssom convert $< -O rdf -o $@
+
+# Merge Mondo, precise mappings and mondo-ingest into one coherent whole for the purpose of querying.
+tmp/merged.owl: tmp/mondo.owl mondo-ingest.owl tmp/mondo.sssom.ttl
+	$(ROBOT) merge -i tmp/mondo.owl -i mondo-ingest.owl -i tmp/mondo.sssom.ttl  --add-prefixes config/context.json -o $@
+
 tmp/merged.db: tmp/merged.owl
 	@rm -f .template.db
 	@rm -f .template.db.tmp
 	RUST_BACKTRACE=full semsql make $@
 	@rm -f .template.db
 	@rm -f .template.db.tmp
 
-../mappings/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-sources-all-lexical.py
-	python $^ run tmp/merged.db -c metadata/mondo.sssom.config.yml -r config/mondo-match-rules.yaml -o $@
+../mappings/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-sources-all-lexical.py tmp/merged.db
```

Comment:
@hrshdhgd This here already ensures that `tmp/merged.db` is recreated!

Reply:
```
tmp/merged.db: tmp/merged.owl
	@rm -f .template.db
	@rm -f .template.db.tmp
	RUST_BACKTRACE=full semsql make $@
	@rm -f .template.db
	@rm -f .template.db.tmp
```
needs to be 
```
tmp/merged.db: tmp/merged.owl
	@rm -f .template.db
	@rm -f .template.db.tmp
	RUST_BACKTRACE=full semsql make $@ -P config/prefixes.csv
	@rm -f .template.db
	@rm -f .template.db.tmp
```

Reply:
But these are all built-in prefixes - it should never be necessary to pass these in?

Reply:
You're right...